src/charm_state.py

```diff
@@ -105,24 +93,24 @@ class Config:  # pylint: disable=too-few-public-methods
 
         extra = Extra.allow
 
-    @validator("smtp_notif_from", pre=True, always=True)
+    @validator("notif_from", pre=True, always=True)
     @classmethod
-    def set_default_smtp_notif_from(
-        cls, smtp_notif_from: typing.Optional[str], values: dict
+    def set_default_notif_from(
```

Comment:
Missed this during the initial review, would this function better be named something like
```suggestion
    def get_default_notif_from(
```
since it doesn't quite set any class attributes?